### PR TITLE
Modularization-main bugfixes

### PR DESCRIPTION
**Pull request by dbalintx**
PR to fix issues on modularization-main

- removing duplicated SSM parameter from container stack (issue came from merging of main)
- adding missing imports

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by nikpodsh — APPROVED** [review]

**Review by nikpodsh — COMMENTED** [review]
The PR is expecting to have more commits. I'll take a look once more then all changes are here

**Comment by dlpzx** [comment]
I added a small fix related to a missing KMS key #613 in my [latest commit ](https://github.com/awslabs/aws-dataall/pull/604/commits/b3bbfe274bf266dfb4be6a2853bd91a938ce86bc)

**Comment by dlpzx** [comment]
Added V1.6.1 fixes in [commit](https://github.com/awslabs/aws-dataall/pull/604/commits/aa1559d1a0ef3c4e115705b9cd3fe5510aa678c8)

File: backend/dataall/core/stacks/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-from dataall.core.stacks import api
+from dataall.core.stacks import api, handlers
```
Comment (dbalintx) [inline backend/dataall/core/stacks/__init__.py:1]: shouldn't we include here `tasks` too?

Comment (nikpodsh) [reply backend/dataall/core/stacks/__init__.py:1]: It is not necessary. The tasks are references by the fullname

**Comment by nikpodsh** [comment]
LGTM!

**Pull request merged**